frontend/src/services/index.js

```diff
@@ -1,13 +1,12 @@
 import axios from "axios";
 
 const API = axios.create({
-	baseURL: "http://127.0.0.1:9203/ditrit/Gandalf/1.0.0",
+	baseURL: process.env.BASEURL,
 });
 
-if (localStorage.getItem("user")) {
+localStorage.getItem("user") &&
```

Comment:
If think you can make:
```js
if (localStorage.getItem("user") {
  API.interceptors.request.use(...);
  API.interceptors.response.use(...);
}
```
to avoid redundant check on localStorage

Reply:
Sure, That 's what i did, i'think that i did not push the wrigth version.
![Caption](https://user-images.githubusercontent.com/7687826/170940180-31fb3e78-cce0-40b6-8ffb-7c43bb210013.png)